src/images/BikeIcon.js

```diff
@@ -4,7 +4,7 @@ export default function BikeIcon() {
   return (
     <svg
       id="Capa_1"
-      enable-background="new 0 0 511.996 511.996"
+      enableBackground="new 0 0 511.996 511.996"
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/enable-background is deprecated